cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -291,9 +292,16 @@ protected void configure() {
       authorizerInstantiator.get().grant(NamespaceId.DEFAULT, principal, ImmutableSet.of(Action.ADMIN));
     }
     namespaceAdmin = injector.getInstance(NamespaceAdmin.class);
-    namespaceAdmin.create(NamespaceMeta.DEFAULT);
+    if (firstInit) {
```

Comment:
Yeah the startCount counts nested invocations, such as happens when run inside a TestSuite, the suite may extend from TestBase, and every Test will also extend from TestBase. Perhaps that should be renamed to nestedStartCount?